Headers/DebugServer2/Utils/HexValues.h

```diff
@@ -42,7 +42,7 @@ template <typename T> static inline std::string ToHex(T const &vec) {
   std::string result;
   for (char n : vec) {
     result += NibbleToHex(n >> 4);
-    result += NibbleToHex(n & 15);
+    result += NibbleToHex(n & 0x0f);
```

Comment:
I think the goal here is to just be a little bit more explicit about what we're doing. Having a uint4_t would be cute but I'm not sure I want to implement that right now :p